### PR TITLE
Update theme blue to accessible alternative

### PR DESCRIPTION
**Pull request by craigpalermo**
In order to meet [WCAG 2.0 AA standards for color contrast](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html), we're updating our `primary.base` color from `#007aff` to `#0068EF`.

### CODE REVIEW

File: packages/core/src/Chip/Chip.spec.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react'
 import { fireEvent, render, screen } from 'testing-library'
 import { act, renderHook } from '@testing-library/react-hooks'
+import { theme } from '..'
```
Comment (achoboter) [inline packages/core/src/Chip/Chip.spec.js:4]: non-blocking, so this doesn't have to be updated in the future :P

```suggestion
import { createTheme } from '..'

const theme = createTheme()
```

File: packages/core/src/Chip/Chip.spec.js [hunk 2]
```diff
@@ -49,7 +50,7 @@ describe('Chip', () => {
 
     expect(Component).toHaveStyleRule('border', '1px solid #c0cad5')
     expect(Component).toHaveStyleRule('background-color', '#fff')
-    expect(Component).toHaveStyleRule('color', '#007aff')
+    expect(Component).toHaveStyleRule('color', theme.colors.blue)
```
Comment (achoboter) [inline packages/core/src/Chip/Chip.spec.js:53]:
```suggestion
    expect(Component).toHaveStyleRule('color', theme.palette.primary.base)
```

File: packages/core/src/Chip/Chip.spec.js [hunk 3]
```diff
@@ -60,9 +61,9 @@ describe('Chip', () => {
     )
     const Component = screen.getByText('Free Wifi')
 
-    expect(Component).toHaveStyleRule('border', '1px solid #007aff')
+    expect(Component).toHaveStyleRule('border', `1px solid ${theme.colors.blue}`)
```
Comment (achoboter) [inline packages/core/src/Chip/Chip.spec.js:64]:
```suggestion
    expect(Component).toHaveStyleRule('border', `1px solid ${theme.palette.primary.base}`)
```

File: packages/core/src/Chip/Chip.spec.js [hunk 4]
```diff
@@ -60,9 +61,9 @@ describe('Chip', () => {
...
     expect(Component).toHaveStyleRule('background-color', '#e8f2ff')
-    expect(Component).toHaveStyleRule('color', '#007aff')
+    expect(Component).toHaveStyleRule('color', theme.colors.blue)
```
Comment (achoboter) [inline packages/core/src/Chip/Chip.spec.js:66]:
```suggestion
    expect(Component).toHaveStyleRule('color', theme.palette.primary.base)
```

**Review by achoboter — APPROVED** [review]

**Review by sdalonzo — APPROVED** [review]

**Pull request merged**